style.css

```diff
@@ -0,0 +1,139 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Tahoma, Geneva, Verdana, sans-serif;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+header {
+  width: 100vw;
+  height: 100vh;
+  background-color: #000;
+  background-image: linear-gradient(62deg, #3a3d40 0%, #181719 100%);
+}
+
+#navbar {
+  display: flex;
+  width: 100vw;
+  justify-content: flex-end;
+  position: fixed;
+  background-color: #be3144;
+}
```

Comment:
![originalcopy (2)](https://user-images.githubusercontent.com/71140133/145084463-052e4a93-2da6-44a4-8daf-3297d132386f.PNG)

![portfolio nav](https://user-images.githubusercontent.com/71140133/145084568-95d87505-104c-4583-a6f0-6e88fdd4889a.PNG)

- I kindly suggest you make your design conform to the original design as much as possible. Include the hamburger menu on the left side of the navbar in your design.